components/SearchSandbox/containers/Editor.js

```diff
@@ -249,7 +249,55 @@ export default class Editor extends Component {
 						</Card>
 
 						<Card>
-							<SelectedFilters />
+							<SelectedFilters
+								render={(props) => {
+									const { selectedValues, setValue, clearValues } = props;
+									const clearFilter = (component) => {
+										setValue(component, null);
+									};
+
+									const filters = Object.keys(selectedValues).map((component) => {
+										if (
+											!selectedValues[component].value
+											|| selectedValues[component].value.length === 0
+										) return null;
+										const value = `${component} : ${
+											selectedValues[component].value
+										}`;
+
+										return (
+											<div
```

Comment:
You can simply import reactivesearch button from here:

```
import Button from '@appbaseio/reactivesearch/lib/styles/Button';
```

instead of writing manual styles.